global.json

```diff
@@ -1,16 +1,30 @@
 {
   "sdk": {
-    "version": "8.0.303",
+    "version": "9.0.100-preview.7.24407.12",
     "rollForward": "major",
     "allowPrerelease": true
   },
   "tools": {
-    "dotnet": "8.0.303"
+    "dotnet": "9.0.100-preview.7.24407.12",
+    "runtimes": {
+      "dotnet/x64": [
+        "8.0.8"
+      ],
+      "dotnet/arm64": [
+        "8.0.8"
```

Comment:
Do we have to have this duped by architecture? Can we do what efcore does?

https://github.com/dotnet/efcore/blob/ecfee78eb1fa2b2eaa0dbf945f1d4f8fa571be74/global.json#L9-L12